From a9d744360c368b4b8adb0c83f6d036cecf8ac450 Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Wed, 26 Jul 2017 17:56:51 +0300
Subject: [PATCH 2225/5242] arm64: dts: freescale: Fix asrc1 node address

commit  508f15df13e392930408369aedebb09fabf06cc7 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to DMA_Audio_8X_v0.5.04.docx, ASRC1
is mapped at 0x59000000 + 0x800000 = 0x59800000.

Most likely, the current value is a copy/paste error
from ASRC0.

Fixes: cc20b6b242 ("MLK-15317-5: ARM64: dts: Add asrc1 node definition")
Reported-by: Viorel Suman <viorel.suman@nxp.com>
Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 3ae1c67..9bddb74 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1576,7 +1576,7 @@
 		status = "disabled";
 	};
 
-	asrc1: asrc@59000000 {
+	asrc1: asrc@59800000 {
 		compatible = "fsl,imx8qm-asrc1";
 		reg = <0x0 0x59800000 0x0 0x10000>;
 		interrupts = <GIC_SPI 380 IRQ_TYPE_LEVEL_HIGH>,
-- 
1.7.9.5

